include/ntptimeclient.h

```diff
@@ -39,7 +39,8 @@
 #include <mutex>
 
 // define the NTP server to connect too (replace . [dots] in IP addresses with , [commas])
-#define cszNTPServer  94, 199, 173, 123     // 0.pool.ntp.org
+#define cszNTPServer  192, 168, 1, 2
+//#define cszNTPServer  94, 199, 173, 123     // 0.pool.ntp.org

```

Comment:
This will make the NTP setup tailored to your specific local environment. Does it make sense to make that the premise for the public (version of the) repository?